src/llama-hparams.h

```diff
@@ -169,6 +169,15 @@ struct llama_hparams {
     uint32_t laurel_rank  = 64;
     uint32_t n_embd_altup = 256;
 
+    // needed for sentence-transformers dense layers
+    uint32_t dense_2_feat_in = 0; // in_features of the 2_Dense
```

Comment:
Nit: Can you lines these up like the other fields in this struct.

Reply:
@sfallah Sorry I was not clear about what I meant. I mean like in https://github.com/ggml-org/llama.cpp/blob/c5fef0fcea3b978a2318b1af170209ecec7c37b4/src/llama-hparams.h#L166-L170 where the fields lineup. Like this:
```c++
    // needed for sentence-transformers dense layers
    uint32_t dense_2_feat_in  = 0; // in_features of the 2_Dense
    uint32_t dense_2_feat_out = 0; // out_features of the 2_Dense
    uint32_t dense_3_feat_in  = 0; // in_features of the 3_Dense
    uint32_t dense_3_feat_out = 0; // out_features of the 3_Dense
```